drivers/include/net/netdev2.h

```diff
@@ -74,6 +74,9 @@ typedef enum {
     NETDEV2_EVENT_RX_COMPLETE,      /**< finished receiving a packet */
     NETDEV2_EVENT_TX_STARTED,       /**< started to transfer a packet */
     NETDEV2_EVENT_TX_COMPLETE,      /**< finished transferring packet */
+#ifdef MODULE_OPENTHREAD
+    NETDEV2_EVENT_TX_COMPLETE_DATA_PENDING, /**< finished transferring packet and has data pending flag **/
```

Comment:
From what I see this and `NETDEV2_EVENT_TX_COMPLETE` are two distinct event types, so why were they throw in the first place? @daniel-k according to the log you introduced this in https://github.com/RIOT-OS/RIOT/pull/3729. Any memory of why those were not separated?